example/android/app/src/debug/AndroidManifest.xml

```diff
@@ -4,7 +4,4 @@
          to allow setting breakpoints, to provide hot reload, etc.
     -->
     <uses-permission android:name="android.permission.INTERNET"/>
-    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
```

Comment:
@mumin-khan In LexActivator-Android we add two permissions, one for "`android.permission.INTERNET`" and the other one for the "`<uses-permission android:name="com.google.android.providers.gsf.permission.READ_GSERVICES" />`"

Reply:
@ahmad-kemsan done.